bot/exts/evergreen/latex.py

```diff
@@ -44,14 +46,18 @@ def _render(text: str, filepath: pathlib.Path) -> BytesIO:
 
         Saves rendered image to cache.
         """
-        fig = plt.figure()
+        fig = figure.Figure()
         rendered_image = BytesIO()
         fig.text(0, 1, text, horizontalalignment="left", verticalalignment="top")
 
         try:
-            plt.savefig(rendered_image, bbox_inches="tight", dpi=600)
+            fig.savefig(rendered_image, bbox_inches="tight", dpi=600)
         except ValueError as e:
             raise commands.BadArgument(str(e))
+        finally:
+            fig.clf()
+            del fig
+            gc.collect()
```

Comment:
I don't think we should be manually messing with memory, especially the global python GC. Scaleios is working on a mutliprocessing version of this cog, so that we can limit the memory of the spawned process.

This can be removed until that is ready.